src/geometamaker/utils.py

```diff
@@ -0,0 +1,28 @@
+import yaml
+
+
+def represent_str(dumper, data):
+    scalar = yaml.representer.SafeRepresenter.represent_str(dumper, data)
+    if len(data.splitlines()) > 1:
+        scalar.style = '|'  # literal style, newline chars will be new lines
+    return scalar
+
+
+# Patch the default string representer so that it uses
+# a literal block style when the data contain newline characters
+yaml.SafeDumper.add_representer(str, represent_str)
+
+
+# https://stackoverflow.com/questions/13518819/avoid-references-in-pyyaml
+class _NoAliasDumper(yaml.SafeDumper):
+    """Keep the yaml human-readable by avoiding anchors and aliases."""
+
+    def ignore_aliases(self, data):
+        return True
```

Comment:
Since we are subclassing `_NoAliasDumper` from `yaml.SafeDumper`, is `self.add_representer` something that we could or should call in an `__init__(...)` method of `_NoAliasDumper`?  It makes sense to me that we would want to modify behavior, but I guess I'm just wondering if we can do this without modifying global state, which would put us at risk of a race condition that could break expected behavior.